tesla_ver/layout.py

```diff
@@ -17,71 +17,100 @@
-                                        ]),
-
-                                html.Div([
-                                    html.P('X Axis'),
-                                    dcc.Dropdown(
-                                        id='x_dropdown',
-                                        options=[],
-                                        value=None,
-                                        placeholder='X Axis Values',
-                                        className='dropdowns'
-                                        ),
-                                        ]),
-
-                                html.Div([
-                                    html.P('Sizing Values'),
-                                    dcc.Dropdown(
-                                        id='size_dropdown',
-                                        options=[],
-                                        value=None,
-                                        placeholder='Sizing Values',
-                                        className='dropdowns'
-                                        ),
-                                        ]),
-
-                                html.Div([
-                                    html.P('Annotation'),
-                                    dcc.Dropdown(
-                                        id='annotation_dropdown',
-                                        options=[],
-                                        value=None,
-                                        placeholder='Annotation',
-                                        multi=True,
-                                        className='dropdowns'
-                                        )
-                                    ]),
-                              ])
-                          ]),
-
+        html.Div(
+            id="graph",
+            # All graph components should go here
+            children=[
+                # Provides an empty graph object for updates in callback
+                dcc.Graph(id="graph-with-slider"),
+                # Creates Slider from min-max X values to give input for graph updates
+                html.Div(
+                    [
+                        dcc.Slider(
+                            id="year-slider",
+                            min=0,
+                            max=1,
+                            value=None,
+                            marks={},
+                            updatemode="drag",
+                        )
+                    ]
+                ),
+                html.Div(
+                    [
+                        html.P("Y Axis"),
+                        dcc.Dropdown(
+                            id="y_dropdown",
+                            options=[],
+                            value=None,
+                            placeholder="Y Axis Values",
+                            className="dropdowns",
+                        ),
+                    ],
+                    style={
```

Comment:
In Issuefix-tesla-ver_UI we removed styling from layout.py. Have you merged the latest changes from Master into your branch?

Reply:
addressed in 04d6a12